docs/linux.md

```diff
@@ -39,15 +39,15 @@ Curl version 7.86
 	yum remove curl libcurl-devel
 	```
 
-#### Steps to Upgrade the OpenSSL* Toolkit to Version 1.1.1s
+#### Steps to Upgrade the OpenSSL* Toolkit to Version 3.0.7
```

Comment:
Can we mention here that alternatively one can run the script here to install openssl n curl?